en/02_Developer_Guides/01_Templates/03_Requirements.md

```diff
@@ -256,6 +259,7 @@ SilverStripe\View\Requirements_Backend:
   combine_in_dev: true
   combine_hash_querystring: true
   default_combined_files_folder: 'combined'
+  resolve_relative_css_refs: true
```

Comment:
I don't think this is needed since it's true by default?

Reply:
Actually that's a good point it probably shouldn't be true by default for BC reasons... can you please amend the docs above to say "you can set this to true to enable it"? I'll note on the other PR to change default to false.

Once it's merged, you can create a PR targetting `5` to set it to true in the next major

Reply:
I changed the last sentence to "Disabled by default"